package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "digitalmarketplace-frontend-toolkit",
   "description": "A toolkit for design patterns used across Digital Marketplace projects",
-  "version": "28.13.2",
+  "version": "28.13.3",
```

Comment:
I reckon this merits at least a minor bump, it's a whole new feature for the toolkit :-)

Reply:
fair point 👍